flight/Modules/Stabilization/stabilization.c

```diff
@@ -941,13 +976,25 @@ static void SettingsUpdatedCb(UAVObjEvent * ev)
 		// The dT has some jitter iteration to iteration that we don't want to
 		// make thie result unpredictable.  Still, it's nicer to specify the constant
 		// based on a time (in ms) rather than a fixed multiplier.  The error between
-		// update rates on OP (~300 Hz) and CC (~475 Hz) is negligible for this
-		// calculation
+		// update rates on OP (~300 Hz) and CC (~475 Hz) is negligible for this calculation
+		//fakeDt now only used for vbar_decay, for gyro filter now a real, but filtered, dT is used
 		const float fakeDt = 0.0025f;
```

Comment:
As i don't have any experience with this flight-mode i didn't want to touch it, till someone thinks it should be changed.
But i will do that.

Should also all text be removed, or just modified to have some history documentation in there.